src/main/java/sg/ncl/MainController.java

```diff
@@ -1358,7 +1358,7 @@ public String teams(Model model, HttpSession session) {
 //        return "redirect:/teams";
 //    }
 
-    @RequestMapping("/withdraw/{teamId}")
+//    @RequestMapping("/withdraw/{teamId}")
     public String withdrawnJoinRequest(@PathVariable Integer teamId, Model model, HttpSession session) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "model". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)